fossunited/foss_profiles/doctype/foss_user_profile/foss_user_profile.json

```diff
@@ -294,21 +294,25 @@
    "print": 1,
    "read": 1,
    "report": 1,
-   "role": "System Manager",
+   "role": "Administrator",
    "share": 1,
    "write": 1
   },
   {
-   "create": 1,
+   "delete": 1,
    "email": 1,
    "export": 1,
-   "print": 1,
    "read": 1,
    "report": 1,
-   "role": "All",
-   "select": 1,
-   "share": 1,
-   "write": 1
+   "role": "System Manager"
+  },
+  {
+   "read": 1,
+   "role": "Guest"
+  },
+  {
+   "read": 1,
+   "role": "All"
```

Comment:
does this mean that everyone can read a profile but not do a get all? why the need for two separate "Guest" and "All" with "read"?

Reply:
Yes, that is the idea. 

All counts for Registered Users, Guest for Unregistered users.